packages/manager/modules/account-sidebar/src/user-infos/template.html

```diff
@@ -53,7 +59,7 @@
     <p data-ng-if="$ctrl.me">
         <span
             class="d-block manager-hub-user-infos_text-small"
-            data-ng-if="$ctrl.me.organisation"
+            data-ng-if="$ctrl.role !== 'provider' && $ctrl.me.organisation"
```

Comment:
related to : https://github.com/ovh/manager/pull/6406/files?file-filters#r798573120